pkg/securityscan/controller.go

```diff
@@ -153,3 +165,49 @@ func detectKubernetesVersion(ctx context.Context, k8sClient kubernetes.Interface
 	}
 	return v.GitVersion, nil
 }
+
+func initializeMetrics(ctl *Controller) error {
```

Comment:
Need to add docs explaining our design decisions for metrics, something similar to https://github.com/kubernetes/kube-state-metrics/blob/00085a19b08a94d938d439377190e4ef3700c622/docs/endpoint-metrics.md.

Reply:
Should be covered in  the docs issue for all CIS v2 enhancements: https://github.com/rancher/docs/issues/2848